src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -96,7 +96,7 @@ public static ECKeyPair fromSeed(byte[] seed) {
 		}
 
 		try {
-			return fromPrivateKey(Hash.hash256(seed));
+			return fromPrivateKey(HashUtils.sha256(seed).asBytes());
```

Comment:
Perhaps it worth to add something like fromPrivateKeyHash() method which will accept HashCode directly?